subworkflows/local/call_structural_variants.nf

```diff
@@ -17,9 +19,11 @@ workflow CALL_STRUCTURAL_VARIANTS {
     main:
         ch_versions = Channel.empty()
 
-        CALL_SV_MANTA( bam, bai, fasta, fai, case_info, target_bed )
+        CALL_SV_MANTA ( bam, bai, fasta, fai, case_info, target_bed )
         ch_versions = ch_versions.mix(CALL_SV_MANTA.out.versions)
 
+        TIDDIT_SV ( bam, fasta, fai )
```

Comment:
@jemten @ramprasadn This isn't a subwrkflw & still thinking if I should put it in its own file but for now should we leave as is since the tiddit/sv workflow is fairly simple?

Reply:
I don't think it has to be a subworkflow if we only need to do a single call to the tiddit_sv module and no processing of the channels. We can alway move it laster

Reply:
I agree @jemten